src/compiler/third-party-out-of-tree.md

```diff
@@ -0,0 +1,168 @@
+with [the GitHub documentation for repository transfers][repo_transfers] and then arrange to perform
+the transfer. Once a transfer is complete, a redirect will exist in the original account or
+organization and this will conflict with the names of any new forks of the transferred repository -
+an email to GitHub is required to resolve this.
+
+[repo_transfers]: https://help.github.com/en/articles/transferring-a-repository
+
+### Who owns these crates?
+It is desired that a compiler team member or working group has loose ownership over a crate so
+that there are clear owners who are responsible for making sure that new versions are published and
+that pull requests are reviewed.
+
+### What should these crates be named?
+Crate naming will be discussed when new out-of-tree crates are proposed to the compiler team.
+
+Crate naming will differ on a case-by-case basis. Crates that are inherently tied to the
+compiler would benefit from a name that is prefixed with `rustc_`. This is an indicator of how
+stable the crate may be to prospective users. Other crates, which are more general-purpose, will
+have names that are disentangled from the compiler.
+
+### Are there any limitations on the review policy for out-of-tree crates?
+Generally, the working groups and team members that are primarily free to maintain the crate using
+whatever practices are best suited to their group, however, there are some limitations so that there
+is some uniformity across the compiler and out-of-tree crates:
+
+- Everyone with r+ on `rust-lang/rust` should be able to review and approve PRs.
+- Where possible, only active participants in the crate (or related working group) need be on the
+  highfive rotation for the crate.
+- It is fine to have additional reviewers on the crate who do not otherwise have r+ for Rust as a
+  whole, if those reviewers are actively involved in the working group or crate maintenance.
+- Major pull requests should have multiple reviewers.
+
+### What is required of an out-of-tree crate?
+It is required that out-of-tree crates must:
+
+- Be dual-licensed with Apache 2.0 and MIT when maintained by the compiler team (as the compiler
+  is) unless there is a compelling reason to do otherwise.
+    - If another license is desired, this must be brought up when proposing the new crate for
+      compiler team members to agree. Prefer [licenses accepted by tidy][licenses], unless otherwise
+      required (ports of code from other projects, etc).
+- Abide by Rust's code of conduct.
+- Specify that the crate is maintained by the Rust compiler team and any appropriate working groups.
+    - In particular, this should detail the expected level of maintenance and stability for any
+      prospective users.
+    - This should also link to the working group details in this repository.
+- Be added to the list at the bottom of this page.
+- Follow semantic versioning.
+- Use `@bors` and `@rust-highfive`.
```

Comment:
I think this has changed to merge queues recently.

Reply:
Fixed